api/main.py

```diff
@@ -192,3 +193,64 @@ def root():
     except KeyError as e:
         print("KeyError: ", e)
         raise HTTPException(status_code=400)
+
+
+@app.get("/files/{file_name}/graph")
+async def get_graph_for_file(
+    file_name: str,
+    score: int = 0,
+    par_length: int = 0,
+    co_occ: int = 0,
+    limit_collection: List[str] = Query([]),
+):
+    try:
+        language = get_language_from_filename(file_name)
+        db = get_db()
+        query_graph_result = db.AQLQuery(
+            query=query_graph_data,
+            bindVars={
+                "filename": file_name,
+                "score": score,
+                "parlength": par_length,
+                "coocc": co_occ,
+            },
+        )
+        parallel_count = 0
+        collection_keys = []
+        total_collection_key_list = []
+        for parallel in query_graph_result.result:
+            parallel_count += 1
+            collection_key = re.search(
+                r"^(pli-tv-b[ui]-vb|[A-Z]+[0-9]+|[a-z\-]+)", parallel
```

Comment:
can we extract this regex to a constant with a descriptive name?

Reply:
I guess we can. It is used a number of times throughout.